opflexagent/utils/ep_managers/endpoint_file_manager.py

```diff
@@ -779,7 +780,7 @@ def _dissociate_port_from_es(self, port_id, port_mac=None, ess=None):
             es_list = self.es_port_dict.keys()
         else:
             es_list = ess
-        for es in es_list:
+        for es in list(es_list):
```

Comment:
Wonder why do we need to put list() around those returned by keys() API here.....The keys() API should already return a list even in python3 according to this,
https://www.tutorialspoint.com/python3/python_dictionary.htm